AsyncDisplayKit/ASCellNode.h

```diff
@@ -14,6 +14,15 @@ NS_ASSUME_NONNULL_BEGIN
 
 typedef NSUInteger ASCellNodeAnimation;
 
+typedef enum : NSUInteger {
+  // Indicates a cell has just became visible
+  ASCellNodeVisibilityEventVisible,
```

Comment:
Maybe DidBecomeVisible?  I wonder if we should drop Visibility from these names, as we may even use ASCellNodeEvent for other things.  Perhaps we should call it ASCellInterfaceEvent or something...or have custom extensions to ASInterfaceState that only apply to ASCellNode.  Dunno, just random ideas, not sure yet what to do hehe.  Directionally though this patch looks like a big improvement.  We do have several clients of this though so I want to make sure to get it right.